src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -64,19 +65,26 @@ const ProteinsListItemInfo = (props: Props) => {
     null
   );
 
+  const proteinId =
+    transcript.product_generating_contexts[0].product.unversioned_stable_id;
+
   useEffect(() => {
     const abortController = new AbortController();
-    const proteinId =
-      transcript.product_generating_contexts[0].product.unversioned_stable_id;
+    fetchProteinDomains(proteinId, abortController.signal).then(
+      (proteinDomains) => {
+        if (!abortController.signal.aborted) {
+          setTranscriptWithProteinDomains(
+            addProteinDomains(transcript, proteinDomains)
+          );
+        }
+      }
+    );
 
-    Promise.all([
-      fetchProteinDomains(proteinId, abortController.signal),
-      fetchProteinSummary(proteinId, abortController.signal)
-    ]).then(([proteinDomains, proteinSummaryData]) => {
+    fetchProteinSummary(
+      transcript.unversioned_stable_id,
```

Comment:
`fetchProteinSummary` function now expects protein id. Changed in this PR: https://github.com/Ensembl/ensembl-client/commit/29b8f1cd5e83ac0bf63879b8d2ce3f1f86809bd7. Please check how the id should be passed to this function.

Reply:
That's done